### ISSUE TITLE
2314 A desync

### ISSUE THREAD

**Issue by GoogleFrog** [beyond-all-reason/spring#1369]
Replay pending

**Comment by GoogleFrog** [comment]
Here is mine

[The Rock Jungle 2.0.pe2-a81a10b9.zip](https://github.com/beyond-all-reason/spring/files/14626189/The.Rock.Jungle.2.0.pe2-a81a10b9.zip)
[The Rock Jungle 2.0.pe-a81a10a9.zip](https://github.com/beyond-all-reason/spring/files/14626190/The.Rock.Jungle.2.0.pe-a81a10a9.zip)
[2024-03-17_03-37-05-494_The Rock Jungle 2_105.1.1-2314-g9e0bf7d BAR105.sdfz.txt](https://github.com/beyond-all-reason/spring/files/14626192/2024-03-17_03-37-05-494_The.Rock.Jungle.2_105.1.1-2314-g9e0bf7d.BAR105.sdfz.txt)

The map is too large to upload: http://zero-k.info/Maps/Detail/64223

Game link: http://zero-k.info/Battles/Detail/1854662

**Comment by GoogleFrog** [comment]
And I don't know where or what the new debug stuff is.

**Comment by Mankarse** [comment]
I desynced near the start of this game.
I reconnected, and then desynced again.

Before I desynced the first time, my screen went black for a few seconds a couple of times. (This may or may not be related).

https://github.com/Mankarse/Zero-K-Debug/releases/download/untagged-8deb47713446b03490bd/2024-03-17-MankarseDesync.zip

**Comment by Mankarse** [comment]
[ClientGameState--749245531-[3461-3461].txt](https://github.com/beyond-all-reason/spring/files/14626755/ClientGameState--749245531-.3461-3461.txt)
[ClientGameState--1903798685-[3461-3461].txt](https://github.com/beyond-all-reason/spring/files/14626756/ClientGameState--1903798685-.3461-3461.txt)

These are both my ClientGameState files (I guess one is from my first desync, and the other is from when i reconnected and desynced again).

**Comment by GoogleFrog** [comment]
[ClientGameState--653912092-[3461-3461].txt](https://github.com/beyond-all-reason/spring/files/14626768/ClientGameState--653912092-.3461-3461.txt)

**Comment by Beherith** [comment]
Diff the clientgamestate files to find out what.

**Comment by lhog** [comment]
@Mankarse run the replay and attach the infolog and the new client state file it's going to generate for you.
I have a suspicion your copy of the game assets is not exactly same as GoogleFrog's.

The log shows not only the desynced unit itself, but multiple issues loading unit textures, that GoogleFrog's PC loaded just fine.

**Comment by Mankarse** [comment]
Thanks for looking at this @lhog 

I ran the replay again today. Here are the infolog and GameState files from my execution of the replay:

[Mankarse_Replay_ServerGameState--318936590-[3461-3461].txt](https://github.com/beyond-all-reason/spring/files/14646451/Mankarse_Replay_ServerGameState--318936590-.3461-3461.txt)
[Mankarse_Replay_infolog.txt](https://github.com/beyond-all-reason/spring/files/14646454/Mankarse_Replay_infolog.txt)

In my previous ClientGameState files:

This new ServerGameState file matches the ClientGameState from GoogleFrog, with the exception of some textureType differences. (That is: my original ClientGameState had some differences for the coordinates of a shieldbomb

**Comment by lhog** [comment]
Yeah, there was something temporary on your side during that desync run and now it's gone.

**Comment by Mankarse** [comment]
Strange that the temporary problem persisted even when I reconnected to the game.

Should I try also restarting my computer next time?
Are there any caches that I should clear before reconnecting to fix a desync?
